### PR TITLE
Adding malachai artifice replica support

### PR DESCRIPTION
**Pull request by uilman**
Adding malachai artifice original stub; needs work to allow the config options to appear, but otherwise is correct

### CODE REVIEW

**Comment by uilman** [comment]
There are *some* potential side effects; with the ring equipped, you always see the config options for EE/ alt ailments. I personally believe this is easier to use because you don't need to swap to your proccing skill in order to change those settings, but it is inconsistent with other things

File: Modules/ModParser-3_0.lua [hunk 1]
```diff
@@ -1582,6 +1582,27 @@ local specialModList = {
 	["socketed gems reserve no mana"] = { mod("ManaReserved", "MORE", -100, { type = "SocketedIn", slotName = "{SlotName}" }) },
 	["socketed skill gems get a (%d+)%% mana multiplier"] = function(num) return { mod("ExtraSkillMod", "LIST", { mod = mod("SupportManaMultiplier", "MORE", num - 100) }, { type = "SocketedIn", slotName = "{SlotName}" }) } end,
 	["socketed gems have blood magic"] = { flag("SkillBloodMagic", { type = "SocketedIn", slotName = "{SlotName}" }) },
+	["socketed gems have elemental equilibrium"] = function() 
+		--Currently hardcoded to what Elemental Equilibrium node is
+		local plus = 25
```
Comment (ppoelzl) [inline Modules/ModParser-3_0.lua:1587]: I think it is fine to hardcode these values without refactoring into variables, like with the other mods.

**Review by ppoelzl — CHANGES_REQUESTED** [review]
I think it is acceptable to always show EE, especially since you'll likely use a different skill than your main skill to proc it.

**Review by ppoelzl — APPROVED** [review]
LGTM now.

**Pull request merged**